src/components/DatePicker/DatePicker.visual.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { visualize, story, snap } from 'storybook-snapper';
+
+import { DatePicker } from './';
+
+visualize('DatePicker', () => {
+  story('simple', () => {
+    snap(
+      'default props',
+      <DatePicker value={new Date('2019/12/15')} onChange={() => {}} />,
+    );
+    snap(
+      'FilterDate - Prior for today dates (includes today)',
+      <DatePicker
+        value={new Date('2019/12/15')}
+        filterDate={date => date <= new Date('2019/12/15')}
+        onChange={() => {}}
+      />,
+    );
+    snap(
+      'FilterDate - feature dates only (includes today)',
+      <DatePicker
+        value={new Date('2019/12/15')}
+        filterDate={date => date >= new Date('2019/12/15')}
+        onChange={() => {}}
+      />,
+    );
+    snap(
+      'Months dropdown',
+      <DatePicker
+        value={new Date('2019/12/15')}
+        showMonthDropdown
+        onChange={() => {}}
+      />,
+    );
+    snap(
+      'Years dropdown',
+      <DatePicker
+        value={new Date('2019/12/15')}
+        showYearDropdown
+        onChange={() => {}}
+      />,
+    );
+    snap(
+      'Months and years dropdown',
+      <DatePicker
+        value={new Date('2019/12/15')}
+        showMonthDropdown
+        showYearDropdown
+        onChange={() => {}}
+      />,
+    );
+  });
```

Comment:
Let's add a visual test for a selected day, and for the focus-ring

Reply:
It is already tested:)
The focus is added automatically on the selected day so the focus ring is displayed on all the visual tests.